### PR TITLE
SSE-RESTEasy for EE9

### PR DESCRIPTION
**Pull request by jim-krueger**

### CODE REVIEW

**Comment by jim-krueger** [comment]
#build

**Comment by jim-krueger** [comment]
#build

**Comment by jim-krueger** [comment]
#build

File: dev/com.ibm.ws.jaxrs.2.1.sse_fat/fat/src/com/ibm/ws/jaxrs21/sse/fat/DelaySseTest.java [hunk 1]
```diff
@@ -20,6 +20,7 @@
 import com.ibm.websphere.simplicity.ShrinkHelper;
 
 import componenttest.annotation.Server;
+import componenttest.annotation.SkipForRepeat;
```
Comment (andymc12) [inline dev/com.ibm.ws.jaxrs.2.1.sse_fat/fat/src/com/ibm/ws/jaxrs21/sse/fat/DelaySseTest.java:23]: can you update the copyright date for this file?

File: dev/com.ibm.ws.jaxrs.2.1.sse_fat/fat/src/com/ibm/ws/jaxrs21/sse/fat/BasicSseTest.java [hunk 2]
```diff
@@ -20,6 +20,7 @@
 import com.ibm.websphere.simplicity.ShrinkHelper;
 
 import componenttest.annotation.Server;
+import componenttest.annotation.SkipForRepeat;
```
Comment (andymc12) [inline dev/com.ibm.ws.jaxrs.2.1.sse_fat/fat/src/com/ibm/ws/jaxrs21/sse/fat/BasicSseTest.java:23]: can you update the copyright date for this file?

**Review by andymc12 — APPROVED** [review]
LGTM - but before you merge, can you update the copyright date for a couple of files?  Thanks!

File: dev/com.ibm.ws.jaxrs.2.1.sse_fat/fat/src/com/ibm/ws/jaxrs21/sse/fat/DelaySseTest.java (hunk 1 above)
Comment (jim-krueger) [reply dev/com.ibm.ws.jaxrs.2.1.sse_fat/fat/src/com/ibm/ws/jaxrs21/sse/fat/DelaySseTest.java:23]: Done

File: dev/com.ibm.ws.jaxrs.2.1.sse_fat/fat/src/com/ibm/ws/jaxrs21/sse/fat/BasicSseTest.java (hunk 2 above)
Comment (jim-krueger) [reply dev/com.ibm.ws.jaxrs.2.1.sse_fat/fat/src/com/ibm/ws/jaxrs21/sse/fat/BasicSseTest.java:23]: Done

**Comment by jim-krueger** [comment]
#run-libby-bot

**Pull request merged**